docs/api/paths/public/db-_session.yaml

```diff
@@ -23,68 +25,30 @@ post:
   description: |-
     Generates a login session for the user based on the credentials provided in the request body or if that fails (due to invalid credentials or none provided at all), generates the new session for the currently authenticated user instead. On a successful session creation, a session cookie is stored to keep the user authenticated for future API calls.
 
-    If CORS is enabled, the origin must match an allowed login origin otherwise an error will be returned.
+    If `Origin` header is passed to this endpoint, the `Origin` header must match both the `cors.login_origin` and `cors.origin` configuration options.
   requestBody:
-    description: The body can depend on if using the Public or Admin APIs.
+    description: The body can be used to override the credentials of the authenticating user, or blank if getting a user session for the authenticating user.
```

Comment:
```suggestion
    description: When name and password are included in the request body, the session will be created for the specified user.  Otherwise the session will be created for the authenticated user making the request.
```